testapps/testapp/src/main/AndroidManifest.xml

```diff
@@ -56,9 +57,17 @@
             </intent-filter>
         </activity>
 
+        <activity
+            android:name="com.microsoft.identity.client.testapp.MainActivity"
+            android:theme="@style/AppTheme.NoActionBar"
+            android:windowSoftInputMode="stateHidden"
+            android:taskAffinity=""
+            android:documentLaunchMode="always">
+        </activity>
```

Comment:
No intent filter here?

Reply:
No.  It's just need for the activity that you want to be used by the launcher.